torchtext/experimental/transforms.py

```diff
@@ -451,7 +451,7 @@ def _random_masking(self, tokens: torch.tensor, mask_prob: float) -> torch.Tenso
         batch_size, seq_len = tokens.size()
         num_masked_per_seq = int(seq_len * mask_prob)
 
-        mask = torch.zeros((batch_size, seq_len), dtype=torch.int)
+        mask = torch.zeros((batch_size, seq_len), dtype=torch.int).to(tokens.device)
```

Comment:
@parmeet do we know if the `to(tokens.device)` is required here? AFAIK all of the text preprocessing happens on CPU right now so there would never be a need to move the tensor to GPU (since `tokens.device` should always be `cpu`). Does this make sense?